.github/pipeline-descriptor.yml

```diff
@@ -9,7 +9,7 @@ codeowners:
 package:
   repository:     docker.io/buildpacksio/profile
   register:       true
-  registry_token: ${{ secrets.GITHUB_TOKEN }}
+  registry_token: ${{ secrets.IMPLEMENTATION_GITHUB_TOKEN }}
```

Comment:
@dmikusa-pivotal should it be using the implementation token to publish to dockerhub? Shouldn't it be some dockerhub credential?

Reply:
There are too many registries, lol. This is referring to the buildpacks registry. The docker creds are below, line 16 & 17.

This change in the descriptor file, results in the change on line 223 in the pb-create-package.yml file below.

I think/hope this will allow us to register the profile buildpack on https://registry.buildpacks.io. It didn't work when I tried to use the GITHUB_TOKEN.